src/components/cards/FeedsCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { View, Text, StyleSheet, TouchableOpacity, Image, TouchableHighlight } from "react-native";
+import React from "react";
+import { colors } from "../../constants/Colors";
+import { fonts } from "../../assets/fonts/fonts";
+import { formatDate } from "../../utils/formatTime";
+import FeedIcon from "../../assets/icons/FeedIcon";
+
+interface FeedCardProps {
+  body: string;
+  image: string | null | undefined;
+  created_at: string;
+}
+
+const FeedsCard = ({ body, image, created_at }: FeedCardProps) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>{body}</Text>
+      <Image
+        source={{
+          uri: image,
+        }}
+        style={styles.image}
+      />
+      <View style={styles.post_details}>
+        <TouchableHighlight style={styles.flex_row}>
+          <Text style={styles.share_btn}>Share</Text>
+          {/* <FeedIcon style={styles.feed_icon} /> */}
```

Comment:
Add the correct share icon based on the [design](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/aabbe326-f4d9-4fae-9cbc-433491567613/specs/).
- Download the icon in SVG format from the design
- Convert the SVG icon into a component using the link https://react-svgr.com/playground/?native=true&typescript=true
- Save it in the assets/icons folder. You can also check out how the other icon components have been done.